### ISSUE TITLE
Several questions :)

### ISSUE THREAD

**Issue by Khaled00777** [euMts/Sickomode#1]
Thanks for this project, dude. I was looking for it very long time but could not find it anywhere but here. Appreciate your work.

I have several silly questions, hope you would not mind :)

1. If 1 want to add 5 html file then is the .json file will look like this?
`{
    "ssid":"Free WiFi",
    "page":"pages/index.html"
    "page":"pages/index2.html" 
    "page":"pages/index3.html"
    "page":"pages/index4.html"
    "page":"pages/index5.html"
}`

2. Which html page will open in captive portal.

3. Can I link 1 html file with other?

4. I want a index page what have not any form and link other html page in that file. Is that possible?? Example A sign in page what only have facebook, google, insta, twitter login option button but not any email-password input option. 
![IMG_20240513_101137](https://github.com/euMts/Sickomode/assets/169632832/65d9f3c2-d5a5-43d3-b524-28fad32344e2)

5. Feature request: add ftp server to control .josn file via file manager

**Comment by euMts** [comment]
Im glad to hear.
At the moment the system only uses one html page at time, unfortunately your json will not work.
The html locatin is set up on line 109 of the [captive_portal.ino](https://github.com/euMts/Sickomode/blob/main/captive_portal/captive_portal.ino) file, it gets the string value passed to the json key and then looks up for the filename stored in the SD card.
Yes, you can link one html file with other, but, you'll need to write an specific code for this to work, in your use case, I would make it as simple as possible, everything in one html file and changing the page style accordingly to wich button the user clicks.
Im currently overworked with other projects, but ill try my best to take a look and launch an update as soon as possible.

Ps. Remember to use with caution, too many uses can result in legal problems, it is intended to study only.

**Comment by Khaled00777** [comment]
Bro I tried to use your default index page to change ssid and page path.. By this code.. But couple of problem that I faced..

1.What will the value of data="" In void handleSettings().

2. The config.json must be clear before run handleSetting() how this could be done?

2. If it change the json file,  then how the esp8266 will restart because we need a restart..

I have not good idea in coding but tried my best, I hope your " config.json editing from web" feature helped by this edit a little bit..
The code-
[captiveportal.txt](https://github.com/euMts/Sickomode/files/15448370/captiveportal.txt)

Settings.json file-
`{
    "path":"pages/settings.html"
}`
